web/src/app/util/OtherActions.js

```diff
@@ -16,71 +15,61 @@ const cancelable = (_fn) => {
   return cFn
 }
 
-export default class OtherActions extends React.PureComponent {
-  static propTypes = {
-    actions: p.arrayOf(
-      p.shape({
-        label: p.string.isRequired,
-        onClick: p.func.isRequired,
-      }),
-    ).isRequired,
-    color: p.string,
-    icon: p.element,
-    placement: p.oneOf(['left', 'right']),
-  }
+export default function OtherActions({ color, icon, ref, actions, placement }) {
```

Comment:
`ref` is not a prop for `OtherActions`, which is causing `make check` to fail. You will likely have to use something like `useRef` hook to replicate `this.ref`

More info here: https://reactjs.org/docs/hooks-reference.html#useref